dll/win32/netcfgx/lang/es-ES.rc

```diff
@@ -41,7 +41,7 @@ BEGIN
     LTEXT "Escriba a continuación la configuración IP alternativa si este equipo  se utiliza en más de una red.", -1, 9, 9, 220, 20
     CONTROL "&Dirección IP privada automática", IDC_USEDHCP, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 14, 40, 210, 12
     GROUPBOX "", -1, 9, 55, 228, 130, BS_GROUPBOX
```

Comment:
Why not using the height of 125 for this groupbox as well. Did you saw anything overflowing into that groupboxes border for es-ES.rc? Can you share a pic how it would look with 125 height?

Reply:
Fixed.